src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -121,6 +117,10 @@
     /** Compiled regexp to match Javadoc tags with no argument and {}. */
     private static final Pattern MATCH_JAVADOC_NOARG_CURLY =
             CommonUtil.createPattern("\\{\\s*@(inheritDoc)\\s*\\}");
+    /** Compiled regexp to match Javadoc tags with argument but with mussing description. */
+    private static final Pattern MATCH_JAVADOC_ARG_MISSING_DESCRIPTION =
```

Comment:
Why was this moved outside it's original position at line 102?
If you want thing in alpabetical order, shouldn't it be there?

Reply:
Yes, you right. Rearranged code.